.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,14 @@
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## Steps to Test or Reproduce
+Steps to reproduce the behavior:
```

Comment:
I personally find it easiest to see the issue if people link an example repo. Most reproduction steps aren't very accurate.
Could we ask for that?

Also we should ask what version of rswag the user is on as that is often a good tip off people aren't up to date.